docker-compose/database/db-changes/datasets/0031_create_processing_resource.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `processing_resource` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `resource_id` varchar(255) NOT NULL,
```

Comment:
Is it necessary to have an id and a ressource_id ?

Reply:
`id` is the PK of the table, while `resource_id` is the UUID associated with one or multiple `dataset_id`.
It's true that a PK here is not really necessary, however as I extends `AbstractEntity` for my `ProcessingResource` entity I don't really have a choice (?).